pom.xml

```diff
@@ -1531,6 +1531,17 @@ xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xs
                       <allowedImport>org.rocksdb.TransactionLogIterator.BatchResult</allowedImport>
                       <allowedImport>org.rocksdb.TickerType</allowedImport>
                       <allowedImport>org.rocksdb.LiveFileMetaData</allowedImport>
+                      <allowedImport>org.rocksdb.AccessHint</allowedImport>
+                      <allowedImport>org.rocksdb.BuiltinComparator</allowedImport>
+                      <allowedImport>org.rocksdb.ChecksumType</allowedImport>
+                      <allowedImport>org.rocksdb.CompactionPriority</allowedImport>
+                      <allowedImport>org.rocksdb.CompactionStopStyle</allowedImport>
+                      <allowedImport>org.rocksdb.CompressionType</allowedImport>
+                      <allowedImport>org.rocksdb.DataBlockIndexType</allowedImport>
+                      <allowedImport>org.rocksdb.IndexShorteningMode</allowedImport>
+                      <allowedImport>org.rocksdb.IndexType</allowedImport>
+                      <allowedImport>org.rocksdb.PrepopulateBlobCache</allowedImport>
+                      <allowedImport>org.rocksdb.WALRecoveryMode</allowedImport>
```

Comment:
Could we move the new classes to `hadoop-hdds/managed-rocksdb/...` instead of adding `allowedImport`?

Reply:
@szetszwo  Thanks for your comments,  I have updated the code to fully support configuring all rocksDB parameters via ini files